interfaces/automated-testing/scd/scd.yaml

```diff
@@ -36,6 +36,25 @@ components:
           enum: [Starting, Ready]
           example: Ready
 
+    InjectFlightResponse:
+      type: object
+      required:
+        - results
+      properties: 
+        result: 
+          description: >-
+            The result of the flight submission
+
+              - `Planned`: The flight submission data was correct and the operational intent was successfully accepted by the DSS.
```

Comment:
nit: I think the USS can check whether flight submission data is "valid", but it can't really check if it's "correct".

Also, I think we can be a bit more general in the success criteria.  I would recommend instead: `The flight submission data was valid and the flight was successfully processed by the USS and is now authorized.`

Reply:
Thanks! I have now added this.